src/api/base.js

```diff
@@ -1,3 +1,4 @@
+export const defaultUrl = process.env.NODE_ENV === 'development' ?  '' : 'http://localhost:8080';
```

Comment:
Let's rename to `baseUrl`, we're not going to change this value, we use it more as a prefix